wtr-utils.js

```diff
@@ -183,8 +167,11 @@ const testRunnerHtml = (testFramework) => `
 `;
 
 const getScreenshotFileName = ({ name }, type, diff) => {
-  const [component, test] = name.split(':');
-  return path.join('chrome', type, component, diff ? `${test}-diff` : test);
+  const [meta, test] = name.split('_');
```

Comment:
I'm using `_` as a delimiter here because this character is not used in URLs produced by `@web/test-runner`.